### PR TITLE
Add development version of coq-trakt.

### PR DESCRIPTION
**Pull request by tomaz1502**

### CODE REVIEW

File: extra-dev/packages/coq-trakt/coq-trakt.dev/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+depends: [
+  "coq-elpi" {>= "2.0.0"}
+  "coq" {>= "8.15~" }
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword:automation"
+  "keyword:elpi"
+  "date:2022-05-16"
```
Comment (silene) [inline extra-dev/packages/coq-trakt/coq-trakt.dev/opam:21]: Having a release date in a package that follows a branch does not make much sense.

File: extra-dev/packages/coq-trakt/coq-trakt.dev/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-trakt"
```
Comment (silene) [inline extra-dev/packages/coq-trakt/coq-trakt.dev/opam:2]: The package name is already part of the filename.

File: extra-dev/packages/coq-trakt/coq-trakt.dev/opam [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+maintainer: [ "Enzo Crance <enzo.crance@inria.fr>" "Louise Dubois de Prisque <louise.dubois-de-prisque@inria.fr>" "Chantal Keller <ckeller@lmf.cnrs.fr>" "Tomaz Mascarenhas <tomgm1502@gmail.com>" ]
+authors: [ "Enzo Crance" ]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/ecranceMERCE/trakt"
+bug-reports: "https://github.com/ecranceMERCE/trakt/issues"
+dev-repo: "git+https://github.com/ecranceMERCE/trakt.git"
+doc: "https://ecrancemerce.github.io/trakt/"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
```
Comment (silene) [inline extra-dev/packages/coq-trakt/coq-trakt.dev/opam:11]: I see that the development comes with some Elpi files. Do they need to be installed? Are they?

Comment (tomaz1502) [reply extra-dev/packages/coq-trakt/coq-trakt.dev/opam:10]: They do not need to be installed. The documentation of Elpi explains that it stores the information in the corresponding .vo file (https://lpcic.github.io/coq-elpi/tutorial_coq_elpi_command.html)

**Comment by silene** [comment]
Could you please rebase on top of the `master` branch to get the latest changes to `coq-core.8.19.dev`, so that the continuous integration has a chance to run?

**Pull request merged**